process explorer/backend/LocalCollector/Connections/CurrentConnections.cs

```diff
@@ -0,0 +1,41 @@
+﻿/* Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. See the NOTICE file distributed with this work for additional information regarding copyright ownership. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. */
+
+namespace ProcessExplorer.Entities.Connections
+{
+    public class ConnectionMonitor 
+    {
+        public List<IConnection> connections;
+        public ConnectionMonitor()
+        {
+            connections = new List<IConnection>();
+        }
+        public void AddConnection(ref IConnection connectionInfo)
```

Comment:
For one of my following steps, i want to check if the status has been changed and trigger an event or something like that, but i've deleted the ref keyword, because that's not the right way.